### PR TITLE
Improve ILL loaders documentation

### LINKED ISSUE #34226: Improve documentation of ILL loaders
The ILL team has received a number of complaints from new users regarding the documentation of dedicated loaders. It was found lacking in terms pointing the user whether a specialised or a general loader should be used to load a number of files, and sometimes the documentation seemed incomplete or confusing. 

- [ ] Add references to Load and `LoadAndMerge` algorithms to ILL's specialised loaders
- [ ] Clear out any inconsistencies and incomplete descriptions of specialised loaders' behaviour

ILL specialised loaders include: `LoadILLDiffraction`, `LoadILLSANS`, `LoadILLIndirect2`, `LoadILLReflectometry`, `LoadILLTOF2`, `LoadILLPolarizedDiffraction`.  

**Additional context**

### PR DESCRIPTION
**Pull request by darominski**
The ILL team has received a number of complaints from new users regarding the documentation of dedicated loaders. It was found lacking in terms pointing the user whether a specialised or a general loader should be used to load a number of files, and sometimes the documentation seemed incomplete or confusing.  This PR aims to fix the following issues:

- [ ] Add references to Load and `LoadAndMerge` algorithms to ILL's specialised loaders
- [ ] Clear out any inconsistencies and incomplete descriptions of specialised loaders' behaviour

ILL specialised loaders include: `LoadILLDiffraction`, `LoadILLSANS`, `LoadILLIndirect2`, `LoadILLReflectometry`, `LoadILLTOF2`, `LoadILLPolarizedDiffraction`.  

**To test:**
1. Documentation clarity and formatting

Fixes #34226. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/algorithms/LoadILLReflectometry-v1.rst [hunk 1]
```diff
@@ -32,7 +36,8 @@ Measurement
 The loader can load both types of data: direct and reflected beam.
 In both cases the foreground centre will be fitted.
 In case of direct beam, the detector will be rotated around the sample such that the fractional workspace index of the foreground centre will appear at 0 scattering angle.
-In case of reflected beam, **BraggAngle** is mandatory, and the detector will be driven such that the foreground centre will appear at **2*BraggAngle**.
+In case of reflected beam, if **BraggAngle** is provided, then it will be the angle used for the rotation of the detector such that the foreground centre will appear at **2*BraggAngle**.
+If the **BraggAngle** property is not specified, then the sample angle metadata position (`SAN`)` is going to be used instead.
```
Comment (jhaigh0) [inline docs/source/algorithms/LoadILLReflectometry-v1.rst:40]:
```suggestion
If the **BraggAngle** property is not specified, then the sample angle metadata position (`SAN`) is going to be used instead.
```

Tiny typo

File: docs/source/algorithms/LoadILLIndirect-v2.rst [hunk 2]
```diff
@@ -9,13 +9,19 @@
...
+This loader can load only a single file at each call. If loading more than one file is required, please refer to
+:ref:`Load <algm-Load>` or :ref:`LoadAndMerge <algm-LoadAndMerge>` algorithms, which are more suited to that task.
+
+
+The loader supports both QENS and FWS types of data, both with or without mirror sense.
 The number of active single detectors, and their actual positions are read from the `.nxs` file overriding the ones in the instrument definition.
 The output workspace will be dimensionless containing the number of counts per energy channel for each detector pixel.
 It will have `empty` x-axis unit.
+
 The `LoadDetectors` allows to choose which data to load for IN16B: those acquired by the spectrometer,
```
Comment (jhaigh0) [inline docs/source/algorithms/LoadILLIndirect-v2.rst:22]:
```suggestion
The **LoadDetectors** option allows to choose which data to load for IN16B: those acquired by the spectrometer,
```

To make refferencing this function parameter in-line with the change made to ``LoadILLTOF-v2``

**Review by jhaigh0 — CHANGES_REQUESTED** [review]
This is still listed as a draft but feels as if its in its final version.

Good changes which read well and the inclusion of links is nice. Just a couple very small formatting / typo changes.

**Comment by perenon** [comment]
Hi @jhaigh0.

This PR is really old and is now totally outdated. Do you agree if I delete it and come back later with a fresh and updated new one ?

**Pull request closed without merging**